HACKING.md

```diff
@@ -38,69 +38,37 @@ that require a cluster to properly test your code.
 
 In those situations, you will need to:
 
-1) build the NTO image you're interested in testing
+1) Build the NTO image you're interested in testing
 2) Push the image to an OpenShift cluster
 
-## Build NTO image
-
-To build an image that contains all NTO components run:
+Before continue with the next steps, please change the following environment variables in the **makefile**: 
```

Comment:
No, users shouldn't need to change makefile variables at all.  You can call make with something like:

```
make <target> VARIABLE=xyz
```